jsonnet/kube-prometheus/prober/probe.libsonnet

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
Are you sure this is what you wanted to commit? As title just says "Bump kube-rbac-proxy to latest version" :) I would prefer if this was in a separate PR, otherwise nice! :)

Reply:
I somehow missed this :facepalm: Should be fixed now.